universal-application-tool-0.0.1/Gruntfile.js

```diff
@@ -0,0 +1,41 @@
+module.exports = function (grunt) {
+  grunt.initConfig({
+    watch: {
+      files: ['app/views/**/*.java'],
+      tasks: ['default'],
+    },
+  })
+
+  grunt.loadNpmTasks('grunt-contrib-watch')
+  grunt.registerTask(
```

Comment:
Alternately, it might be clearer to make grunt the only way to run tailwinds, and just keep all the logic on calling it in one place (here)

Reply:
A totally optional alternative: looks like there's a way to do this just in SBT (using sbt-web), but there's minimal-to-no docs on it.  It would be cool, though!  Maybe something for the future

https://github.com/sbt/sbt-web

One example:
https://github.com/sbt/sbt-uglify/blob/02f0e99a58fc9cefb268278379f1bfce490a87e6/src/main/scala/com/typesafe/sbt/uglify/SbtUglify.scala